<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Todo</string>
  </property>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton{
	background-color: rgb(170, 255, 127);
	color: indigo;
	border-radius: 20px;
}

#pushButton:hover{
	background: rgb(170, 200, 127);
	color: #3ff;
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2{
	background-color: rgb(00, 255, 127);
	color: indigo;
	border-radius: 20px;
}

#pushButton_2:hover{
	background: rgb(255, 000, 000);
	color: #3ff;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Todo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Todo Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Label:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="todoNameTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="todoLabelTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
